mesh11sd/Makefile

```diff
@@ -17,7 +17,7 @@ PKG_LICENSE_FILES:=LICENSE
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/opennds/mesh11sd/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=8029a0e41487d322b1bb20df05fc3ef2073b1239977f2b430b20fe7cabfe71de
+PKG_HASH:=80406e70fadf58320ec4891c8fb1e93d118d2927de7c6d9749a15da6768b4ea5
 PKG_BUILD_DIR:=$(BUILD_DIR)/mesh11sd-$(PKG_VERSION)
```

Comment:
I know that it is not related to your PR, apologies for nitpicking, but isnt this already default?

Reply:
@BKPepe 
Default? Sorry, I don't follow.
But it is a new hash.... Does not the build system calculate the hash from the downloaded source tar.gz and compare what it gets with this value?
I test building the package using the SDK and it requires this to be updated.

Reply:
This is what I mean:

```suggestion
```

Reply:
I see! Are you sure it is a default?

Shall we try it and see? It's Saturday, lets live dangerously :-D

Reply:
Yup, its default https://github.com/openwrt/openwrt/blob/66177c081f1b05e4eb7f6c23d97954e8172c4336/include/package.mk#L11 , no need to specify the same here. :-)

Reply:
> The best line of code is no line of code